dist/profile/templates/buildmaster/casc/global-libraries.yaml.erb

```
@@ -0,0 +1,19 @@
+unclassified:
+  globalLibraries:
+    libraries:
+      - defaultVersion: "master"
+        implicit: true
+        includeInChangesets: false
+        name: "pipeline-library"
```

you're missing the caching config from the other file?
```suggestion
        name: "pipeline-library"
        cachingConfiguration:
          refreshTimeMinutes: 180
```

---

> [...] I've omitted the cache setting for now.

:)

---

I'm resolving conversation to allow merging

---

https://github.com/jenkins-infra/helpdesk/issues/2751 for tracking cache